functions.py

```diff
@@ -0,0 +1,192 @@
+''' Shared library of functions for report.py and remind.py
+'''
+
+import os
+import re
+import datetime
+import bugzilla
+from jira import JIRA
+
+
+def generate_html_file(htmlcode, remind=False):
+	''' generates HTML file of reminder
+	'''
+	try:
+		os.makedirs('archive')
+	except FileExistsError:
+		pass
+	if remind:
+		filename = './archive/reminder_{:%m%d%Y_%H:%M:%S}.html'.format(datetime.datetime.now())
+	else:
+		filename = './archive/report_{:%m%d%Y_%H:%M:%S}.html'.format(datetime.datetime.now())
+	with open(filename, 'w') as file:
+		file.write(htmlcode)
+
```

Comment:
I forget what the Python standard is for void functions - should this have a `return None`?

Reply:
Not sure, I believe it's mostly just stylistic preference - I'm fine with whatever method so long as it's consistent across the project

Will let others weigh in on this

Reply:
https://stackoverflow.com/questions/15300550/return-return-none-and-no-return-at-all

Reply:
Done, will be the next patchset